use/posts.js

```diff
@@ -127,10 +129,8 @@ export const usePosts = () => {
       hideLabel: true,
       type: 'textarea',
       placeholder: 'Text (markdown)',
-      tips:
-        'Pitch yourself: Who are you? What do you offer? What do you want?\n\nTips for effective pitch:\n- Uncomplicated: It should be catchy and roll off the tongue\n- Concise: It shouldn’t take more than a minute to say or read\n- Unique: It reflects your skills, goals, and desires\n- Storyline: It covers who you are, what you offer, and where you want to be\n- Appealing: Your elevator pitch is essentially a persuasive sales pitch; the emphasis should be on what you offer',
-      description:
-        'Use [widgets](https://wedance.vip/markdown), including images and videos',
+      tips:t('posts.edit.tips'),
```

Comment:
Please read #70 

I would organise it like this.

```suggestion
      tips:t('post.description.tips'),
```

So that field is a group and it has `label`, `placeholder`, `tips` and not far from each other. It's easier to maintain it as a translator later.